jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -22,6 +22,15 @@
 __all__ = ['CoordsInfo']
 
 
+def _convert_surface_brightness_units(data, from_unit, to_unit):
+    try:
+        quantity = data * u.Unit(from_unit)
+        converted_quantity = quantity.to(u.Unit(to_unit))
+        return converted_quantity.value
+    except u.UnitConversionError:
+        raise ValueError(f"Conversion from {from_unit} to {to_unit} is not possible.")
```

Comment:
can we reuse `jdaviz.utils._eqv_pixar_sr` to get the necessary equivalencies?  @gibsongreen - is there any other useful re-usable functions from your work?

Reply:
The `jdaviz.utils.flux_conversion` function handles the cases of sb <-> sb for all spectra and custom equivalencies weren't necessary to code in that case. Since the image data isn't passing through this function, it would likely be beneficial to have something like this, but maybe moving it to `jdaviz.utils` so that it can be accessed in a more general way the `_eqv_pixar_sr`.

Reply:
Thank you for your input, Gilbert. I've moved the conversion logic to the utils.py file.